### PR TITLE
🤖 API client snapshots update

### PR DESCRIPTION
**Pull request by johangirod**
*This PR is automatically generated by a bot.*
It updates the snapshots of the API clients.

Review carefully the changes to see if there is no regression:
- No changes in attributes names
- No suspect massive changes in the data
- No changes in the structure of the data

If everything looks good, you can merge this PR. 

Otherwise, you should take action to fix the issue, by either: 
- Contact the administration responsible for the API
- Change the client code to adapt to the new data

### CODE REVIEW

File: clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json [hunk 1]
```diff
@@ -32,7 +32,7 @@
           "commune": "PARIS 13",
           "activitePrincipale": "90.01Z",
           "libelleActivitePrincipale": "Arts du spectacle vivant (90.01Z)",
-          "trancheEffectif": "02",
+          "trancheEffectif": "N",
```
Comment (XavierJp) [inline clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json:35]: @HAEKADI this is odd. This is our test snapshot and it tells `trancheEffectif` changed and company became `Unite non employeuse`. 

Yet I do not find the same result on sirene.fr 800329849

Could it be a bug on sirene API ? Or a bug on our side ?

**Review by XavierJp — COMMENTED** [review]
@HAEKADI I feel like our tests spotted smthg odd here

File: clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json (hunk 1 above)
Comment (HAEKADI) [reply clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json:35]: I am confused. Both the api and sirene.fr indicate the same thing. What snapshot are we talking about ?

Comment (XavierJp) [reply clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json:35]: Ok thie issue might lay in  another variable : `caractereEmployeur` is set to `N` for these entities

Comment (XavierJp) [reply clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json:35]: Ok I dont understand, so I will write a mail to Insee team

Comment (XavierJp) [reply clients/_test/clientSearchRechercheEntreprise/_snapshots/search-800329849.json:35]: Insee team answered. 
Ths `caractereEmployeur` and `trancheEffectif` come from two differents sources.
Either one can be the most up to date and we currently dont know which is.

**Comment by XavierJp** [comment]
Not sure this automated PR uis still valid. I prefer to close it and wait for the next one

**Pull request closed without merging**